build/tools/Automation/Python/Models/PythonVersion.cs

```diff
@@ -1,9 +1,24 @@
-﻿namespace Microsoft.Oryx.Automation.Python.Models
```

Comment:
One minor comment I'm seeing from this change: would you quickly confirm that all of the `Automation` project `.cs` files have the license header?